README.md

```diff
@@ -8,7 +8,15 @@ Discord bot for KaoGeek, built with TypeScript
 # Install dependencies
 pnpm install
 
+# Copy .env.example to .env
 cp .env.example .env
+
+# Change node version to 16
+nvm install v16 # if you don't have node 16 installed
```

Comment:
project actually use node 18, you might want to change this

Reply:
Actually, in [package.json](https://github.com/creatorsgarten/kaogeek-discord-bot/blob/cc9df638a75059b5f2983a3392ac1dab3a89071c/package.json#L42), it specifies both v16 and v18.

Reply:
Yap v16 and v18, but i'm use v19 is default.
I and someone must change node version to develop.

Thank you

Reply:
Change to `nvm install`. It will auto detect the version in .nvmrc